schemas/session-replay/_common-shape-wireframe-schema.json

```diff
@@ -0,0 +1,69 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "_common-shape-wireframe-schema.json",
+  "title": "CommonShapeWireframe",
+  "type": "object",
+  "description": "Schema of common properties for ShapeWireframe events type and all its sub - types.",
+  "allOf": [
+    {
+      "$ref": "_common-wireframe-schema.json"
+    },
+    {
+      "required": [
+        "opacity",
+        "cornerRadius"
+      ],
+      "properties": {
+        "opacity": {
+          "type": "number",
+          "description": "The opacity of this wireframe. Takes values from 0 to 1.",
+          "default": 1,
+          "readOnly": true
+        },
+        "cornerRadius": {
+          "type": "number",
+          "description": "The corner radius of this wireframe.",
+          "default": 0,
+          "readOnly": true
+        },
+        "shapeStyle": {
+          "required": [
+            "fillColor"
```

Comment:
Like I mentioned above, in case we mark a field as optional that means that it will default to `null`. In that case we don't need special defaults for it. Would you rather go with optionals for these fields ?

Reply:
On the other hand I get what you're saying, we can have it optional and add the `default` property just for visibility. This way at the player level they will know to which value to default if they can't find this option.